mkosi/__init__.py

```diff
@@ -6552,7 +6553,7 @@ def load_args(args: argparse.Namespace) -> MkosiArgs:
         die("Sorry, --source-file-transfer-final=mount is not supported")
```

Comment:
Missed one here

Reply:
I really thought this one wasn't the case, thanks